neuroanalysis/util/device_config.py

```diff
@@ -0,0 +1,93 @@
+import neuroanalysis.util.mies_nwb_parsing as parser
```

Comment:
Why do you think mies parsing functions don't belong in neuroanalysis? And I guess the bigger question is what is the goal of neuroanalysis vs aisynphys vs optoanalysis?

One way we could organize would be to put all the base classes (Dataset, Recording etc, DatasetLoader) in neuroanalysis, move mies_parsing and MiesDatasetLoader into aisynphys, and move the Analyzer base classes from aisynphys into neuroanalysis. It seems like we should try to only have one-way imports ie, aisynphys imports neuroanalysis, but neuroanalysis doesn't import from aisynphys. Otherwise it will be easy to get caught in import loops. I'm not really sure where optoanalysis fits at all.

Reply:
When I said "Not sure this belongs in neuroanalysis", I was referring to the whole file -- this file seems to be about inferring rig hardware configuration by some combination of serial numbers and timestamps, which works for the opto project but probably not other projects (in the ideal case, the raw experimental data would fully describe its hardware configuration, and it would not be necessary to make such inferences). The _modeling_ part of it could be in neuroanalysis (what standardized API do we use to ask a Dataset about its hardware), but the implementation is project-specific.

Reply:
Ah, sorry, I totally misinterpreted that. I agree that device_config should be somewhere else, I just wasn't quite sure where, so stuck it here for now :)